values.yaml

```diff
@@ -332,17 +332,30 @@ concourse:
 
     ## A second method by which a worker is selected during container placement
     ## that is only used for get and nested check steps.
-    ## Possible values: volume-locality | random | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
+    ## Possible values: random | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
     noInputContainerPlacementStrategy:
 
     ## Instead of specifying a single placement strategy, configure a chain of
     ## them to be applied in order.
     ##
     ## If set, takes priority over noInputContainerPlacementStrategy.
     ##
-    ## Possible values: volume-locality | fewest-build-containers | limit-active-tasks | limit-active-containers | limit-active-volumes
```

Comment:
why volume-locality is removed here?

Reply:
Because volume-locality is only meaningful for `task` and `put` steps that has input volumes., while `check` and `get` steps don't have any input volume.